MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.TimePicker.xaml

```diff
@@ -117,11 +118,17 @@
 						    		   PopupAnimation="Fade"/>
 						    </Grid>
                         </Border>
+                        <Line x:Name="DashedLine" VerticalAlignment="Bottom" Visibility="Hidden"
+                              StrokeThickness="{Binding BorderThickness.Bottom, ElementName=border, Converter={StaticResource DivisionMathConverter}, ConverterParameter=0.75}" StrokeDashArray="1,2.5" StrokeDashCap="Round"
+                              X1="0" X2="{Binding ActualWidth, ElementName=border}" Y1="0" Y2="0" 
+                              Stroke="{TemplateBinding BorderBrush}" Opacity="0.56" />
                         <wpf:Underline x:Name="Underline" Visibility="{Binding Path=(wpf:TextFieldAssist.DecorationVisibility), RelativeSource={RelativeSource TemplatedParent}}"/>
                     </Grid>
                     <ControlTemplate.Triggers>
                         <Trigger Property="IsEnabled" Value="False">
                             <Setter Property="Opacity" TargetName="TemplateRoot" Value="0.56"/>
```

Comment:
I think you should set opacity to "_border_" element. Setting opacity to outer element (Grid called "_TemplateRoot_") will change also "_DashedLine_" appearence (and the Line already has its opacity set to "0.56")